standalone/pherry/src/lib.rs

```diff
@@ -894,6 +899,66 @@ async fn init_runtime(
     Ok(resp)
 }
 
+async fn update_worker_endpoint(
+    pubkey: &Vec<u8>,
+    para_api: &ParachainApi,
+    encoded_versioned_endpoint: Vec<u8>,
+    signature: Vec<u8>,
+    signer: &mut SrSigner,
+    args: &Args,
+) -> Result<()> {
+    chain_client::update_signer_nonce(para_api, signer).await?;
+    let versioned_endpoint = Decode::decode(&mut &encoded_versioned_endpoint[..])
+        .map_err(|_| anyhow!("Decode versioned endpoint failed"))?;
+    let params = mk_params(para_api, args.longevity, args.tip).await?;
+    let ret = para_api
+        .tx()
+        .phala_registry()
+        .update_worker_endpoint(
+            phaxt::khala::runtime_types::sp_core::sr25519::Public(
```

Comment:
```suggestion
            sp_core::sr25519::Public(
```

Reply:
Seems like the rust compiler forces us to use `phaxt::khala::runtime_types::sp_core::sr25519::Public`.